docs/source/intro.rst

```diff
@@ -199,7 +211,7 @@ ELF Manipulation
 
 Stop hard-coding things!  Look them up at runtime with :mod:`pwnlib.elf`.
 
-    >>> e = ELF('/bin/cat')
+    >>> e = ELF('/bin/cat') # doctest: +SKIP
```

Comment:
Why are we skipping this?

Reply:
Because all the follow-up tests were already skipped.